src/AutoRest.CSharp/MgmtTest/Generation/ResourceCollectionTestWriter.cs

```diff
@@ -276,30 +276,17 @@ protected void WriteMethodTest(MgmtClientOperation clientOperation, bool async)
                     }
 
                     WriteTestDecorator();
-                    if (clientOperation.IsPagingOperation(Context))
-                    {
-                        _writer.Append($"public void {testMethodName}{(exampleIdx > 0 ? (exampleIdx + 1).ToString() : "")}()");
-                    }
-                    else
-                    {
-                        _writer.Append($"public {GetAsyncKeyword(async)} {MgmtBaseTestWriter.GetTaskOrVoid(async)} {testMethodName}{(exampleIdx > 0 ? (exampleIdx + 1).ToString() : "")}()");
-                    }
+                    _writer.Append($"public {GetAsyncKeyword(async)} {MgmtBaseTestWriter.GetTaskOrVoid(async)} {testMethodName}{(exampleIdx > 0 ? (exampleIdx + 1).ToString() : "")}()");
```

Comment:
This part `(exampleIdx > 0 ? (exampleIdx + 1).ToString() : "")` is kind of unintuitive, could we refine this to make this more readable?
and BTW we should use `string.Empty` instead of `""`